pkg/kudoctl/cmd/plan.go

```diff
@@ -11,62 +11,84 @@ import (
 func newPlanCmd(out io.Writer) *cobra.Command {
-	newCmd := &cobra.Command{
+	cmd := &cobra.Command{
 		Use:   "plan",
 		Short: "View all available plans.",
 		Long:  `The plan command has subcommands to view all available plans.`,
 	}
 
-	newCmd.AddCommand(NewPlanHistoryCmd())
-	newCmd.AddCommand(NewPlanStatusCmd(out))
+	cmd.AddCommand(NewPlanHistoryCmd())
+	cmd.AddCommand(NewPlanStatusCmd(out))
+	cmd.AddCommand(NewPlanTriggerCmd())
 
-	return newCmd
+	return cmd
 }
 
 // NewPlanHistoryCmd creates a command that shows the plan history of an instance.
 func NewPlanHistoryCmd() *cobra.Command {
 	options := plan.DefaultHistoryOptions
-	listCmd := &cobra.Command{
+	cmd := &cobra.Command{
 		Use:     "history",
 		Short:   "Lists history to a specific operator-version of an instance.",
-		Example: planHistExample,
+		Example: planHistoryExample,
 		RunE: func(cmd *cobra.Command, args []string) error {
 			return plan.RunHistory(cmd, options, &Settings)
 		},
 	}
 
-	listCmd.Flags().StringVar(&options.Instance, "instance", "", "The instance name.")
+	cmd.Flags().StringVar(&options.Instance, "instance", "", "The instance name available from 'kubectl get instances'")
 
-	return listCmd
+	return cmd
 }
 
 //NewPlanStatusCmd creates a new command that shows the status of an instance by looking at its current plan
 func NewPlanStatusCmd(out io.Writer) *cobra.Command {
 	options := &plan.Options{Out: out}
-	statusCmd := &cobra.Command{
+	cmd := &cobra.Command{
 		Use:     "status",
 		Short:   "Shows the status of all plans to an particular instance.",
-		Example: planStatuExample,
+		Example: planStatusExample,
 		RunE: func(cmd *cobra.Command, args []string) error {
 			return plan.Status(options, &Settings)
 		},
 	}
 
-	statusCmd.Flags().StringVar(&options.Instance, "instance", "", "The instance name available from 'kubectl get instances'")
-	if err := statusCmd.MarkFlagRequired("instance"); err != nil {
-		clog.Printf("failed to mark --instance flag as required: %v", err)
+	cmd.Flags().StringVar(&options.Instance, "instance", "", "The instance name available from 'kubectl get instances'")
+	if err := cmd.MarkFlagRequired("instance"); err != nil {
+		clog.Printf("Please choose the instance with '--instance=<instanceName>': %v", err)
 		os.Exit(1)
 	}
 
-	return statusCmd
+	return cmd
+}
+
+// NewPlanTriggerCmd creates a command that triggers a specific plan for an instance.
+func NewPlanTriggerCmd() *cobra.Command {
+	options := &plan.TriggerOptions{}
+	cmd := &cobra.Command{
+		Use:     "trigger",
+		Short:   "Triggers a specific plan on a particular instance.",
+		Example: planTriggerExample,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return plan.RunTrigger(options, &Settings)
+		},
+	}
+
+	cmd.Flags().StringVar(&options.Instance, "instance", "", "The instance name available from 'kubectl get instances'")
+	cmd.Flags().StringVar(&options.Plan, "name", "", "The plan name")
```

Comment:
I think it should be possible to set parameters here as well?

Reply:
Maybe we can leave that for a followup-PR though

Reply:
Currently setting additional parameters when triggering a plan manually is [explicitly forbidden](https://github.com/kudobuilder/kudo/pull/1352/files#diff-c10c73f4220d3a0c71475b7002482b65R235). We can tackle it in a followup PR because this PR is packed enough 😉